Branch-SDK/src/io/branch/referral/Branch.java

```diff
@@ -696,6 +697,7 @@ public static Branch getAutoInstance(@NonNull Context context) {
         customReferrableSettings_ = CUSTOM_REFERRABLE_SETTINGS.USE_DEFAULT;
         boolean isTest = BranchUtil.checkTestMode(context);
         getBranchInstance(context, !isTest, null);
+        new BranchPreinstall().getPreinstallSystemData(context);
```

Comment:
Note that this appears to be doing "File" APIs, and will be run on the UI thread... which may violate some STRICT preferences.  I can't verify this though.

Reply:
Nice catch. It was coming, moved it to the thread.